site/src/analytics-integrations.ts

```diff
@@ -52,7 +52,17 @@ export class AnalyticsIntegrations extends LitElement {
   protected firstUpdated(_changedProperties: PropertyValues) {
     super.firstUpdated(_changedProperties);
 
-    this.getData();
+    const query = new URLSearchParams(window.location.search);
+    this._filter = query.get("search")?.toLowerCase() || "";
+    this.getData().then(() => {
+      if (
+        this._filter !== "" &&
+        (DEFAULT_DOMAINS.includes(this._filter) ||
+          this._integrationDetails[this._filter].quality_scale === "internal")
```

Comment:
This is a bit of a weird check because it assumes it's an exact match which is not what "filter" implies ?

Reply:
This is only on init, so it only applies when using query param

Reply:
Changed it to "domain" to make it clearer